automation-tools/dlrn.md

```diff
@@ -0,0 +1,17 @@
+#  DLRN
+
+- DLRN builds and maintains yum repositories following upstream commits from a Git repo.
+- [ :computer: github.com/softwarefactory-project/DLRN](https://github.com/softwarefactory-project/DLRN), [ :scroll: documentation](https://dlrn.readthedocs.io/en/latest/intro.html)
+- ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/dlrn.svg), [![PyPI](https://img.shields.io/pypi/v/dlrn.svg)](https://pypi.org/project/dlrn/), [ :package: fedora packages](https://src.fedoraproject.org/rpms/dlrn)
+
+:heavy_plus_sign:
+- automatic builds of packages
+- generating of yum repositories
```

Comment:
this sounds exactly like copr: building packages and providing repositories; I'm assuming that running DLRN is not that easy

Reply:
Similar, but not exactly like copr, it can be used for generating per-commmit repos. (It can use copr as a build backend.)

Setup of the database/web-server would be definitely harder than simply using the copr.